models/index.js

```diff
@@ -0,0 +1,24 @@
+const { Model } = require('sequelize');
```

Comment:
The datatypes are never imported. Just add `{ Model, DataTypes }` to your current line 1 to import the model and datatypes at the same time. or create a new line under your current one `const { DataTypes } = require('sequelize');`